docs/reference/esql/functions/kibana/definition/to_long.json

```diff
@@ -16,6 +16,18 @@
       "variadic" : false,
       "returnType" : "long"
     },
+    {
+      "params" : [
+        {
+          "name" : "field",
+          "type" : "counter_long",
```

Comment:
@drewdaemon this is fun - did you know that things annotated with `counter` aren't `long` to ESQL? Before this PR they were unsupported entirely. With this PR they'll have types like this.

Reply:
On our side, we use Javascript types so it's all `number` :D 

But that may change so this is very good to keep in mind. Thanks for the ping (cross ref with https://github.com/elastic/kibana/issues/174710)